utils/jwt.ts

```diff
@@ -9,19 +9,33 @@ export function genJwt(id: string): string {
   return jwt.sign({id}, process.env.TOKEN_SECRET, { expiresIn: '7d' });
 }
 
-export function validateJwt(req: Request, res: Response, next: NextFunction) {
-  const authHeader = req.headers.authorization;
-  const token = authHeader && authHeader.split(' ')[1];
+function validateJwtToken(
+  req: Request, res: Response,
+  next: NextFunction,
+  token: string | undefined,
+  bodyKey: string) {
```

Comment:
my general rule of thumb is if the parameter list exceeds 3, we should through the parameters into an interface

```ts
interface validateJwtOptions {
  req: Request;
  res: Response;
  next: NextFunction;
  token?: string;
  key: string;
}
```

Reply:
ok sounds good